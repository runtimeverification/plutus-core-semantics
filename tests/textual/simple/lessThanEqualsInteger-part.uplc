(program 1.0.0 [ (builtin lessThanEqualsInteger) (con integer 1) ])