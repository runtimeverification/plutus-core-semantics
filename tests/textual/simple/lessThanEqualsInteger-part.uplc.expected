[ (builtin lessThanEqualsInteger) (con integer 1) ]